Vizzuality/half-earth-v3
# Feature/sidebar card refactor

## Sidebar cards
### Description
This PR abstracts the diferent types of sidebard cards into a new component `sidebar-card-content`
This component is used as a `child` of the `sidebar-card-wrapper` component (that is used to provide styles and stack cards when needed).
If no `type` for the content is provided the `default` content is displayed.

### Designs
[inVision prototype](https://projects.invisionapp.com/d/main#/console/21108730/445055729/preview)

`default`

![image](https://user-images.githubusercontent.com/6906348/108825573-7eb81d80-75c3-11eb-894e-f608eddaf22a.png)

`basic`

![image](https://user-images.githubusercontent.com/6906348/108825633-9099c080-75c3-11eb-8de2-421794ed6541.png)

`gradient`

![image](https://user-images.githubusercontent.com/6906348/108825710-a5765400-75c3-11eb-8881-24c90d938520.png)

### Testing instructions
Check the example cards on the `overview` (map) view of the National Report Cards
